tests/transactional/install_updates.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Install Update repos in transactional server
+# Maintainer: qac team <qa-c@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use qam;
+use transactional;
+
+sub run {
+    my ($self) = @_;
+    select_console 'root-console';
+    assert_script_run 'curl http://ca.suse.de/certificates/ca/SUSE_Trust_Root.crt -o /etc/pki/trust/anchors/SUSE_Trust_Root.crt';
```

Comment:
This only works in internal tests

Reply:
Yeah, but this test is for SLE Micro 5.0 maintenance updates which points to internal repos anyway. I could put it in  condition `if sle-micro`

Reply:
Yes, but the test module doesn't indicate that it's SLE only. `is sle_micro` or `unless is_opensuse` should both work.